src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -272,5 +279,704 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
+        //     => checked((byte)(left / right));
+
+        //
+        // IEqualityOperators
+        //
+
+        [RequiresPreviewFeatures]
+        static bool IEqualityOperators<byte, byte>.operator ==(byte left, byte right)
+            => left == right;
+
+        [RequiresPreviewFeatures]
+        static bool IEqualityOperators<byte, byte>.operator !=(byte left, byte right)
+            => left != right;
+
+        //
+        // IIncrementOperators
+        //
+
+        [RequiresPreviewFeatures]
+        static byte IIncrementOperators<byte>.operator ++(byte value)
+            => value++;
+
+        // [RequiresPreviewFeatures]
+        // static checked byte IIncrementOperators<byte>.operator ++(byte value)
+        //     => checked(value++);
+
+        //
+        // IMinMaxValue
+        //
+
+        [RequiresPreviewFeatures]
+        static byte IMinMaxValue<byte>.MinValue => MinValue;
+
+        [RequiresPreviewFeatures]
+        static byte IMinMaxValue<byte>.MaxValue => MaxValue;
+
+        //
+        // IModulusOperators
+        //
+
+        [RequiresPreviewFeatures]
+        static byte IModulusOperators<byte, byte, byte>.operator %(byte left, byte right)
+            => (byte)(left % right);
+
+        // [RequiresPreviewFeatures]
+        // static checked byte IModulusOperators<byte, byte, byte>.operator %(byte left, byte right)
+        //     => checked((byte)(left % right));
+
+        //
+        // IMultiplicativeIdentity
+        //
+
+        [RequiresPreviewFeatures]
+        static byte IMultiplicativeIdentity<byte, byte>.MultiplicativeIdentity => 1;
+
+        //
+        // IMultiplyOperators
+        //
+
+        [RequiresPreviewFeatures]
+        static byte IMultiplyOperators<byte, byte, byte>.operator *(byte left, byte right)
+            => (byte)(left * right);
+
+        // [RequiresPreviewFeatures]
+        // static checked byte IMultiplyOperators<byte, byte, byte>.operator *(byte left, byte right)
+        //     => checked((byte)(left * right));
+
+        //
+        // INumber
+        //
+
+        [RequiresPreviewFeatures]
+        static byte INumber<byte>.One => 1;
+
+        [RequiresPreviewFeatures]
+        static byte INumber<byte>.Zero => 0;
+
+        [RequiresPreviewFeatures]
+        static byte INumber<byte>.Abs(byte value)
+            => value;
+
+        [RequiresPreviewFeatures]
+        static byte INumber<byte>.Clamp(byte value, byte min, byte max)
+            => Math.Clamp(value, min, max);
+
+        [RequiresPreviewFeatures]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        static byte INumber<byte>.Create<TOther>(TOther value)
+        {
+            if (typeof(TOther) == typeof(byte))
+            {
+                return (byte)(object)value;
+            }
+            else if (typeof(TOther) == typeof(char))
+            {
+                return checked((byte)(char)(object)value);
```

Comment:
What options are there for conversion if you don't want it to be checked?

Reply:
Saturating and Truncating below yes but all these have extra checks, is there a non-checked version?